src/index.html

```diff
@@ -15,8 +15,79 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <ul>
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="img"
```

Comment:
The `alt` attribute should describe the image content. It is used by screen readers and when the image cannot be displayed. It should give a clear understanding of the image content. Therefore, 'img' is not a good description. Please, provide a more meaningful description. For example, if it's a company logo, you can write 'Company Logo'.